recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/NIFTI-Imaging/{{ name }}/archive/de2bb84d6219a3671ef7f5b696f17d93276da9eb.tar.gz
```

Comment:
Please use `https://github.com/NIFTI-Imaging/nifti_clib/archive/v2.0.0.tar.gz` instead of a commit tarball.